.changeset/dry-paws-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Move the github utility into its own folder
```

Comment:
This could be a bit confusing to someone reading the changelog I think. Could pretty much just be "restructured internals", or even skip the changeset as there's no functional difference

Reply:
OK, removed!